src/server/raft_client.rs

```diff
@@ -174,7 +174,9 @@ impl Buffer for BatchMessageBuffer {
         let mut msg_size = msg.start_key.len()
             + msg.end_key.len()
             + msg.get_message().context.len()
-            + msg.extra_ctx.len();
+            + msg.extra_ctx.len()
+            // index: 3, term: 2, data tag and size: 3, entry tag and size: 3
+            + 11 * msg.get_message().get_entries().len();
```

Comment:
There are also other fields in `eraftpb::Message` like `to`, `from`, `term`, etc.

Reply:
Yes, but there are at most 128 messages, so the size can be omitted.